### PR TITLE
add diffrence decimal in liquidation price

### PR DESCRIPTION
**Pull request by Pawel-Szydlo**

### CODE REVIEW

**Review by wojciech-cichocki — APPROVED** [review]

File: src/components/Borrow/BorrowTable/BorrowTable.tsx [hunk 1]
```diff
@@ -125,13 +125,14 @@ export const BorrowTable: React.FC<IProp> = ({
...
                     classes={{ tooltip: classes.tooltipNumber }}
-                    title={`${element.deposited} ${element.collateral}`}
+                    title={`${element.deposited.toFixed(7)} ${element.collateral}`}
                     placement='bottom-start'>
                     <Grid>
                       {'~ '}
                       <AnimatedNumber
-                        value={element.deposited}
+                        value={element.deposited.toFixed(6)}
                         formatValue={formatNumbersBorrowTable}
```
Comment (NorbertBodziony) [inline src/components/Borrow/BorrowTable/BorrowTable.tsx:134]: Imo this should be to fixed decimals of assets

File: src/components/Borrow/BorrowTable/BorrowTable.tsx [hunk 2]
```diff
@@ -140,13 +141,14 @@ export const BorrowTable: React.FC<IProp> = ({
...
+                    title={`${element.currentDebt.toFixed(7)} ${element.borrowed}`}
                     placement='bottom-start'>
                     <Grid>
                       {'~ '}
                       <AnimatedNumber
-                        value={element.currentDebt}
+                        value={element.currentDebt.toFixed(6)}
                         formatValue={formatNumbersBorrowTable}
+                        duration={300}
                       />
```
Comment (NorbertBodziony) [inline src/components/Borrow/BorrowTable/BorrowTable.tsx:152]: Same stuff why 7 and 6 ?

File: src/components/Borrow/borrowUtils.tsx [hunk 3]
```diff
@@ -98,9 +98,24 @@ export const calculateLiqPrice = (
       8
     )
   }
+  const difDecimal = 10 ** (assetScaleTo - assetScaleFrom)
+  if (difDecimal < 1) {
+    console.log(liqThreshold.val.toString())
```
Comment (NorbertBodziony) [inline src/components/Borrow/borrowUtils.tsx:103]: leftover

**Review by NorbertBodziony — CHANGES_REQUESTED** [review]

File: src/components/Borrow/BorrowTable/BorrowTable.tsx (hunk 1 above)
Comment (Pawel-Szydlo) [reply src/components/Borrow/BorrowTable/BorrowTable.tsx:134]: ok, changed

**Review by NorbertBodziony — APPROVED** [review]

**Pull request merged**